test/integration/user-management-api/application_plan_features_test.rb

```diff
@@ -105,10 +105,12 @@ def teardown
 
     test 'associate existing feature' do
       feat = FactoryBot.create(:feature, featurable: @provider.default_service)
+      feat2 = FactoryBot.create(:feature, featurable: @provider.default_service)
 
       assert_difference @app_plan.features.method(:count), 2 do
         post admin_api_application_plan_features_path(@app_plan, format: :xml), params: params.merge({ feature_id: feat.id })
-        post admin_api_application_plan_features_path(@app_plan, format: :xml), params: params.merge({ feature_id: feat.id })
+        result = post admin_api_application_plan_features_path(@app_plan, format: :xml), params: params.merge({ feature_id: feat2.id })
```

Comment:
is the `result` variable needed?

Reply:
No, :(

I added it for debugging purposes and forgot to remove it.

Reply:
Fixed!